lib/validator/v3/validator.go

```diff
@@ -233,6 +234,11 @@ func validatePortName(v *validator.Validate, topStruct reflect.Value, currentStr
 	return len(s) != 0 && len(k8svalidation.IsValidPortName(s)) == 0
 }
 
+func validateMustBeNil(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
+	log.Debugf("Validate field must be nil: %s", field.String())
```

Comment:
Clearly the validator code is a very naughty boy, but we should really be using WithField() I think.